open-company/open-company-web
# Inline media picker

#### Review after #853 

Card: https://trello.com/c/PWmoWtjU

To test:
- open Cmail
- type some text
- [x] do you get the inline picker when the new line is empty?
- [x] can you add an image?
- [x] can you add a YouTube video?
- [x] can you add a Vimeo video?
- [x] can you add a Loom video?
- [ ] can you add a live recorded video with Ziggeo?
- [ ] no trouble when the Ziggeo recording ends, for example the inline media pickler is still shown after you added the video?
- [x] can you add an attachment to the post?
- [x] can you insert emojis in all fields?
- [x] can you delete the draft/post with the trash can?
- now switch to fullscreen
- [ ] make sure you can complete all the steps above
- [x] finally try to publish a post
- [x] try to edit a published post
